tidb-cloud/tidb-cloud-sso-authentication.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Single sign-on Authentication
+summary: Learn how to sign in via your Google workspace account and your Github Accounts in TiDB Cloud Web console.
+---
+# SSO(single sign-on) Authentication
+
+This document describes how individual or corporate users can quickly log in to TiDB CLoud Web Console via Googol workspace and GitHub accounts. Since your identity is centrally hosted on the third-party Google and GitHub platforms, TiDB cloud does not support you to modify your account password and enable MFA after logging in TiDB Cloud via SSO authentication.If you want to know about logging in through user and password , please see [Password Authentication](/tidb-cloud/tidb-cloud-password-authentication.md)
+
```

Comment:
Do we need a section for "sign up with Google SSO"

Reply:
we do not have verify SSO email process, if we improve sso signup workflow ,we should add this section